Sources/SwiftDocCUtilities/Action/Actions/PreviewAction.swift

```diff
@@ -192,38 +200,27 @@ extension PreviewAction {
         }
 
         monitor = try DirectoryMonitor(root: rootURL) { _, _ in
-            print("Source bundle was modified, converting... ", terminator: "", to: &self.logHandle)
+            self.print("Source bundle was modified, converting... ", terminator: "")
             self.monitoredConvertTask?.cancel()
             self.monitoredConvertTask = Task {
-                defer {
```

Comment:
Why don't we need this `defer` block any more?

Reply:
From what I can tell it wasn't ever needed. The `DirectoryMonitor` already does a smarter automatic reloading when files are added to or removed from a monitored directory. 

Removing the restart here means that we don't:
- Restart (stop monitoring all files and directories and start monitoring them again) twice when a file is added removed (once here and once in the directory monitor itself).
- Restart (stop monitoring all files and directories and start monitoring them again) when an already monitored file's content changed but no files were added or removed.